client/src/components/Lists/PitchInterests/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, ReactElement } from 'react';
+import { IPitch } from 'ssw-common';
+
+import { FieldTag } from '../..';
+import { useInterests } from '../../../contexts';
+
+interface PitchInterestsProps {
+  pitch: Pick<IPitch, 'topics'>;
```

Comment:
This means you better be passing in something that looks like

```tsx
{ 
    topics: string[]
}
```

Seems more ambiguous to do it this way. Also see my previous comment.